ops/testing.py

```diff
@@ -227,11 +227,9 @@ def begin_with_initial_hooks(self) -> None:
         # Checking if disks have been added
         # storage-attached events happen before install
         for storage_name in self._meta.storages:
-            for storage_index in self._backend.storage_list(storage_name):
-                storage_name = storage_name.replace('-', '_')
```

Comment:
I didn't see `replace('-', '_')` in the new code. Is it not needed anymore?

Reply:
It was never needed here.  The replacement happens automatically via PrefixedEvents when `.on[name]` is called.